src/env/Makefile.mk

```diff
@@ -29,29 +29,29 @@ endif BUILD_FC_BINDING
 
 if INSTALL_MPICXX
 bin_SCRIPTS += src/env/mpicxx
+if BUILD_ABI_LIB
+bin_SCRIPTS += src/env/mpicxx_abi
+endif BUILD_ABI_LIB
```

Comment:
Since now you do the links in post-install scripts, I don't think we need the `_abi` targets in `src/env/` anymore, right?